test/localhost-e2e.test.ts

```diff
@@ -1,87 +1,29 @@
 // Copyright (c) 2023 Cloudflare, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { spawn } from 'node:child_process';
-import fetch from 'node-fetch';
-import { testE2E } from './e2e/issuer';
+import { SELF } from 'cloudflare:test';
+import { it, describe, beforeAll, expect } from 'vitest';
 
-const ISSUER_URL = 'localhost:8787';
+import { requestBatchedTokens, requestSingleToken } from './e2e/e2eUtils';
 
-describe('e2e on localhost', () => {
-	let serverProcess: ReturnType<typeof spawn> | undefined;
+const sampleURL = 'http://localhost-e2e';
```

Comment:
nit

```suggestion
const sampleURL = 'http://localhost-e2e.test';
```

using test TLD which is made for that